core/src/main/java/com/graphhopper/routing/weighting/custom/CustomWeighting.java

```diff
@@ -102,33 +106,37 @@ public CustomWeighting(FlagEncoder baseFlagEncoder, EncodedValueLookup lookup,
         baseVehicleAccessEnc = baseFlagEncoder.getAccessEnc();
         speedCalculator = new SpeedCalculator(baseFlagEncoder.getMaxSpeed(), customModel, baseFlagEncoder.getAverageSpeedEnc(), lookup);
         maxSpeed = speedCalculator.getMaxSpeed() / SPEED_CONV;
+        prioritySensitivity = 1 / (PRIORITY_SENSITIVITY_FACTOR * maxSpeed);
 
         priorityCalculator = new PriorityCalculator(customModel, lookup);
 
-        // unit is "seconds per 1km"
+        // given unit is s/km -> convert to s/m
         distanceInfluence = customModel.getDistanceInfluence() / 1000;
         if (distanceInfluence < 0)
             throw new IllegalArgumentException("maximum distance_influence cannot be negative " + distanceInfluence);
     }
 
     @Override
     public double getMinWeight(double distance) {
-        return distance / maxSpeed + distance * distanceInfluence;
+        return distance / maxSpeed + calcDistanceCosts(distance);
     }
 
     @Override
     public double calcEdgeWeight(EdgeIteratorState edgeState, boolean reverse) {
-        double distance = edgeState.getDistance();
```

Comment:
BTW: the reason I "cached" the distance here is that it gave a minor speed up. (not sure if this is still true)

Reply:
ok I added a todo and lets check this when we have decided on the formula again